ttnn/ttnn/profiler.py

```diff
@@ -0,0 +1,27 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
```

Comment:
I appreciate the effort to take tracy related functions out of `op_profiler.hpp` however this file is missing some of the implementation. 

stop zone needs the stack to make sure it calls end on python calls that have not reached their end yet. This is for supporting the `python -m tracy -p` option. 

I would copy all the relevant functions from `op_profiler` and the relevant globals here instead of writing the file from scratch.